src/vmm/src/arch/x86_64/msr.rs

```diff
@@ -302,25 +304,25 @@ pub enum SetMSRsError {
 pub fn set_msrs(
     vcpu: &VcpuFd,
     msr_entries: &[kvm_msr_entry],
-) -> std::result::Result<(), SetMSRsError> {
-    let msrs = Msrs::from_entries(msr_entries).map_err(SetMSRsError::Create)?;
+) -> std::result::Result<(), SetMsrsError> {
+    let msrs = Msrs::from_entries(msr_entries).map_err(SetMsrsError::Create)?;
     vcpu.set_msrs(&msrs)
-        .map_err(SetMSRsError::Set)
+        .map_err(SetMsrsError::Set)
         .and_then(|msrs_written| {
             if msrs_written as u32 == msrs.as_fam_struct_ref().nmsrs {
                 Ok(())
             } else {
-                Err(SetMSRsError::Incomplete)
+                Err(SetMsrsError::Incomplete)
             }
         })
 }
 
-/// Returns the list of supported, serializable MSRs.
+/// Returns the list of supported and serializable MSR indices.
 ///
 /// # Arguments
 ///
 /// * `kvm_fd` - Structure that holds the KVM's fd.
-pub fn supported_guest_msrs(kvm_fd: &Kvm) -> Result<MsrList> {
+pub fn get_msrs_to_save(kvm_fd: &Kvm) -> Result<MsrList> {
```

Comment:
I think the function expects the caller to use the returned value to use as MSR index list to save / serialize into snapshot, because it filters unserializable MSR indices from supported MSR indices.

Reply:
I'll add `get_msrs_to_dump()` in a PR to add CPU config dump feature.
https://github.com/firecracker-microvm/firecracker/pull/3586/commits/277ded9e925d547042105b3c5faddd26ae3f2396#diff-646931758f35a261e2f848a0970552a6da048816eae0393b597d5830d857fba1L308-L310